addons/web/static/src/js/views/pivot/pivot_view.js

```diff
@@ -76,7 +78,10 @@ var PivotView = AbstractView.extend({
             // the measure should be allowed.  However, be careful if you define
             // a measure in your pivot view: non stored functional fields will
             // probably not work (their aggregate will always be 0).
-
+            if (field.attrs.invisible && py.eval(field.attrs.invisible)) {
```

Comment:
one last change: this should be above the comment block here.  Note that the comment applies to the next if statement, so it should stay together